Modified io_check script to output variable value and change ratio

TanZiTT opened the pull request:
The io_check.r script was modified. If the variables were found different in
original and modified models.  It will print out the variable values of
both models as well as the change ratios (original values/ modified
values). "output_compare.txt" shows the sample format of output.

kbrannan commented on R_scripts/output_compare.txt:
```diff
@@ -4,13 +4,79 @@ Accum.Pasture
+0 0 NA
+0 0 NA
 
 Onsite Pets Submodel Key Variables:
```
Is the comaprison script working for onsite?script

kbrannan commented:
Check my comment on the on-site part of the comparison. Everything else looks great!

TanZiTT commented:
I see.

There's no difference on key variables between original  and modified on-site model. Thus the script printed out a NA matrix.

I will fix that.

---

From: Kevin notifications@github.com
Sent: Monday, November 16, 2015 5:12 PM
To: kbrannan/ODEQ-Bacteria-Model-R
Cc: Zi, Tan
Subject: Re: [ODEQ-Bacteria-Model-R] Modified io_check script to output variable value and change ratio (#12)

Check my comment on the on-site part of the comparison. Everything else looks great!

## 

Reply to this email directly or view it on GitHubhttps://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157187845.

[https://avatars2.githubusercontent.com/u/15020361?v=3&s=400]https://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157187845

Modified io_check script to output variable value and change ratio by TanZiTT · Pull Request #12 · kbrannan/ODEQ-Bacteria-Model-R
The io_check.r script was modified. If the variables were found different in original and modified models. It will print out the variable values of both models as well as the change ratios (origin...
Read more...https://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157187845

kbrannan commented:
I ran the previous version of io_check (pull request 11) and got the following output for the onsite:
Onsite Pets Submodel Key Variables:
bac.onsite.NearStrmStrctFailure.to.stream.load
Accum.RAOCUT
Different Key Variables:
Accum.RAOCUT

Shouldn't we see this change in the latest version of io_check?

TanZiTT commented:
hmm, yes we should. Seems there're bugs either in the older version or the latest version scripts.
Thanks for the QA, I will look into this.

---

From: Kevin notifications@github.com
Sent: Monday, November 16, 2015 7:38 PM
To: kbrannan/ODEQ-Bacteria-Model-R
Cc: Zi, Tan
Subject: Re: [ODEQ-Bacteria-Model-R] Modified io_check script to output variable value and change ratio (#12)

I ran the previous version of io_check (pull request 11) and got the following output for the onsite:
Onsite Pets Submodel Key Variables:
bac.onsite.NearStrmStrctFailure.to.stream.load
Accum.RAOCUT
Different Key Variables:
Accum.RAOCUT

Shouldn't we see this change in the latest version of io_check?

## 

Reply to this email directly or view it on GitHubhttps://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157221025.

[https://avatars2.githubusercontent.com/u/15020361?v=3&s=400]https://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157221025

Modified io_check script to output variable value and change ratio by TanZiTT · Pull Request #12 · kbrannan/ODEQ-Bacteria-Model-R
The io_check.r script was modified. If the variables were found different in original and modified models. It will print out the variable values of both models as well as the change ratios (origin...
Read more...https://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157221025

TanZiTT commented:
Hi Kevin,

I just updated the script to fix the output issue.

I also ran the previous version of io_check.r script but didn’t see difference in key variables in On-site output. I also checked the output compare spreadsheet file that I shared to you long ago. The key variables in on-site outputs were the same and the output of the script is consistent with the results in spreadsheet.

I think the reason we got different results may due to the input files we used are different. The script is working fine and now if there’s no differences in key variables, it won’t print NA matrix anymore.

Now the files at my end are all up to date. Since the pull request is still open, when you merge the pull, you should be able to get the latest working script.

Regards,
Tan

From: Kevin [mailto:notifications@github.com]
Sent: Monday, November 16, 2015 7:39 PM
To: kbrannan/ODEQ-Bacteria-Model-R ODEQ-Bacteria-Model-R@noreply.github.com
Cc: Zi, Tan Tan.Zi@tetratech.com
Subject: Re: [ODEQ-Bacteria-Model-R] Modified io_check script to output variable value and change ratio (#12)

I ran the previous version of io_check (pull request 11) and got the following output for the onsite:
Onsite Pets Submodel Key Variables:
bac.onsite.NearStrmStrctFailure.to.stream.load
Accum.RAOCUT
Different Key Variables:
Accum.RAOCUT

Shouldn't we see this change in the latest version of io_check?

—
Reply to this email directly or view it on GitHubhttps://github.com/kbrannan/ODEQ-Bacteria-Model-R/pull/12#issuecomment-157221025.

kbrannan commented:
Great, thanks!
